scss/components/_buttons.scss

```diff
@@ -406,4 +406,10 @@ $btn-border-width: 4px;
       box-shadow: none;
     }
   }
+}
+
+@media screen and (-ms-high-contrast: active) {
+  .btn-primary {
+    border: 2px solid;
```

Comment:
Why did you choose `2px`?  I expected `$btn-border-width`.

Reply:
Missed to see this repo code, added border width as $btn-border-width.